src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -164,7 +164,7 @@ const pageQuery = gql`
 					hash
 				}
 				plannedExpirationDate
-				lenders(limit: 3) {
+				lenders(limit: 10) {
```

Comment:
I wish this query wasn't part of the loan data and page load...but it'll do for now...